net/dns/resolver/forwarder.go

```diff
@@ -407,7 +454,11 @@ func (f *forwarder) send(ctx context.Context, fq *forwardQuery, dst netaddr.IPPo
 	fq.closeOnCtxDone.Add(conn)
 	defer fq.closeOnCtxDone.Remove(conn)
 
-	if _, err := conn.WriteTo(fq.packet, dst.UDPAddr()); err != nil {
+	udpAddr := &net.UDPAddr{
```

Comment:
I think the UDPAddr method is easier to read than the stutter of the next line.

Revert this part? `dst.UDPAddr()` can just be `ipp.UDPAddr()`, no?